specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/quota.json

```diff
@@ -0,0 +1,1252 @@
+          },
+          {
+            "$ref": "#/parameters/ScopeInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The quota information will be returned in the response with pagination.",
+            "schema": {
+              "$ref": "#/definitions/CurrentQuotaLimitBase"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "QuotaInformation"
+        ],
+        "description": "Create or update the quota limit for the specified resource to the requested value. To update the quota, follow these steps:\n1. Use the GET operation to determine how much quota remains for the specific resource and to calculate the new quota limit. These steps are detailed in [this example](https://techcommunity.microsoft.com/t5/azure-governance-and-management/using-the-new-quota-rest-api/ba-p/2183670).\n2. Use this PUT operation to update the quota limit.",
+        "operationId": "Quota_CreateOrUpdate",
+        "x-ms-examples": {
+          "Quotas_Request_ForCompute": {
+            "$ref": "./examples/putComputeOneSkuQuotaRequest.json"
+          },
+          "Quotas_Request_ForMachineLearningServices_DedicatedResource": {
+            "$ref": "./examples/putMachineLearningServicesQuotaRequestDedicated.json"
+          },
+          "Quotas_Request_ForMachineLearningServices_LowPriorityResource": {
+            "$ref": "./examples/putMachineLearningServicesQuotaRequestLowPriority.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceNameInParameters"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeInPath"
+          },
+          {
+            "name": "createQuotaRequest",
+            "in": "body",
+            "description": "Quota requests payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CurrentQuotaLimitBase"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns the quota request details.",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestOneResourceSubmitResponse"
+            }
+          },
+          "202": {
```

Comment:
> "202" [](http://example.com/codeflow?start=9&length=6)

For PUT 201 is okay. (I know it's a bit confusing since PATCH requires 202). 201 with provisioningState + Location + Azure-AsyncOperation header is recommended.

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put)

Reply:
I will change Put response to 201

Reply:
Updated to 201